src/rust/bitbox02/src/sd.rs

```diff
@@ -30,6 +30,20 @@ pub fn sdcard_inserted() -> bool {
     data.sdcard_inserted.unwrap()
 }
 
+#[cfg(not(feature = "testing"))]
+pub fn sdcard_vfat_formatted() -> bool {
+    unsafe { bitbox02_sys::sd_card_vfat_formatted() }
+}
+
+#[cfg(feature = "testing")]
+pub fn sdcard_vfat_formatted() -> bool {
+    true
+}
+
+pub fn sd_format() -> bool {
```

Comment:
no need for the `sd` prefix, this is in the `sd` module and we don't need prefixes for namespacing like in C.